### DISCUSSION TITLE
Broken videos

### DISCUSSION THREAD

**Discussion by pomboesferico** [aclap-dev/video-downloadhelper#851] [Help and Q&A]
### Your question

Hi
I'm trying to download some videos but the image and sound are distorted despite being set to download in 720p.

What could be causing this?

![image](https://github.com/aclap-dev/video-downloadhelper/assets/172212167/4e093216-1cb3-449c-9b5f-9e4a26b78bb5)

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"141f6b3abdb04d0f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "USERS",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_wide_ui": true,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 10
}

**Comment by GrampaWildWilly** [comment]
I don't work for VDH.  I'm just another user.

You would be greatly helping yourself if you shared the URL of the example you are showing.  But if it's something that requires a registration, don't bother.

You might try the VDH beta.  Click the link at the top of this page & the top of every page of this forum labelled Wiki.  That is the closest thing we have to a user manual.  Down the right side of that page is a list of chapters.  Click the chapter that mentions beta versions.  You will find the VDH beta there.

As pure speculation, I could suggest maybe the content is protected by DRM.  In that case, you can forget about downloading the content with VDH.  There are expensive services that will break DRM protection but VDH has never, does not now, nor will ever break DRM protection.  For more on the subject of DRM, search this forum using "drm" as your search key.
